app/src/main/java/at/roteskreuz/stopcorona/screens/base/DebugViewModel.kt

```diff
@@ -59,50 +54,19 @@ class DebugViewModel(
 
     fun addOutgoingMessageRed() {
         launch {
-            val infectionMessageContent =
-                InfectionMessageContent(MessageType.InfectionLevel.Red, ZonedDateTime.now())
-
-            // DEBUG-TODO 11-Jun-2020 mihbat: Insert debug temporary exposure keys in database
+            // DEBUG-TODO 11-Jun-2020 mihbat: This method was intended for debugging the
+            //        reporting of a red diagnostic.
```

Comment:
Already fixed. Please revert. 
It will give us a merge conflict.

Reply:
updated

Reply:
Still conflicts. I'll resolve the conflict